design/ceph/admission-controller.md

```diff
@@ -1,15 +1,18 @@
 # Admission Controllers in Rook
 
+!!! Note
```

Comment:
Only the official docs have the notes rendered, not the design docs. No need for this header.
```suggestion
```